src/taskmanager.cpp

```diff
@@ -104,7 +104,8 @@ void TaskManager::backupNow(BackupTaskPtr backupTask)
     }
     bTask->setCommand(makeTarsnapCommand(CMD_TARSNAP));
     bTask->setArguments(args);
-    backupTask->setCommand(bTask->command() + " " + bTask->arguments().join(" "));
+    backupTask->setCommand(bTask->command() + " " +
```

Comment:
This modifies the stored command for the resulting Archive object. Let's quote only when displaying the command in ArchiveWidget::updateDetails().